### PR TITLE
Feature/profile questions 2 - fix IU,  + FE logic

### PR DESCRIPTION
**Pull request by pavlo-chernykh**
what was done:
- fixed IU according to requirements (proper error validation)
- now skills, english level and category data taken properly from back-end using custom hook
- front-end logic

### CODE REVIEW

File: apps/freelance/src/pages/ProfileQuestions/model.ts [hunk 1]
```diff
@@ -1,13 +1,17 @@
 import { Moment } from 'moment';
 
-export interface IProfileQuestions1 {
+export interface IProfileQuestions {
   [key: string]: string | number | object | undefined;
```
Comment (lyudad) [inline apps/freelance/src/pages/ProfileQuestions/model.ts]: `[key: string]?: string | number | object;` should work

File: apps/freelance/src/pages/ProfileQuestions/model.ts [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
   available_time: string;
   description: string;
   hourly_rate: number;
   position: string;
   education: educationProps;
   work_history_wrapper: workProps;
+  category: string;
+  english_level: string;
+  other_experience: string;
+  skills: [];
```
Comment (lyudad) [inline apps/freelance/src/pages/ProfileQuestions/model.ts]: array of what type?

File: apps/freelance/src/pages/ProfileQuestions/model.ts (hunk 1 above)
Comment (pavlo-chernykh) [reply apps/freelance/src/pages/ProfileQuestions/model.ts]: fixed

File: apps/freelance/src/pages/ProfileQuestions/model.ts (hunk 2 above)
Comment (pavlo-chernykh) [reply apps/freelance/src/pages/ProfileQuestions/model.ts]: fixed

**Pull request merged**